addon/components/chart-component.js

```diff
@@ -15,8 +15,32 @@ export default Ember.Component.extend(ColorableMixin, ResizeHandlerMixin, {
   horizontalMargin: 30,
   verticalMargin: 30,
 
-  marginRight: Ember.computed.alias('horizontalMargin'),
-  marginLeft: Ember.computed.alias('horizontalMargin'),
+  /**
+   * Optional property to set specific left margin
+   * @type {Number}
+   */
+  horizontalMarginLeft: null,
+
+  /**
+   * Optional property to set specific right margin
+   * @type {Number}
+   */
+  horizontalMarginRight: null,
+
+  /**
+   * Either a passed in value from `horizontalMarginRight`
+   * or the default value from `horizontalMargin`
+   * @type {Number}
+   */
+  marginRight: Ember.computed.or('horizontalMarginRight', 'horizontalMargin'),
```

Comment:
This seems like a bad idea. If `horizontalMarginRight` is `0` this won't do what we want.